packages/fxa-settings/src/lib/null-storage.test.ts

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import NullStorage from './null-storage';
+
+let storage: NullStorage;
+
+beforeEach(function () {
```

Comment:
Could make all of these arrow functions for funzies, along with those functions in `storage.test.ts`.